radix-engine-interface/src/blueprints/pool/one_resource_pool/invocations.rs

```diff
@@ -10,11 +10,13 @@ define_invocation! {
     function_name: instantiate,
     input: struct {
         resource_address: ResourceAddress,
+        owner_rule: OwnerRole,
```

Comment:
The OwnerRole should come first for consistency with other calls which use an OwnerRole